### DISCUSSION TITLE
bug after login and take long time loading tu use our platform

### DISCUSSION THREAD

**Discussion by aimanndaus** [MusheAbdulHakim/Laravel-Smarthr#32] [Q&A]
![Screenshot 2024-08-19 at 17 46 08](https://github.com/user-attachments/assets/881a18e6-050b-493f-ade2-0b35a5fdbd46)
 
this issue regarding loading on screen and freeze to click other  features.
 i follow the the guide for installation and run smoothly but after finish and i want to use all features freeze due loading on screen.
- for this case i already reinstall but got same problem.
- hope full you can help me to fixed and can use properly thanks

**Comment by MusheAbdulHakim** [comment]
can you open the browser console and screenshot the error message?.

**Reply by aimanndaus** [reply]
it's okay i already fixed it by migrate -- fresh thanks for your attention sir. i hopefully you always update this repo because interested on your project. have a good days sir

**Reply by MusheAbdulHakim** [reply]
ok great.
